LeadKit.xcodeproj/project.pbxproj

```diff
@@ -2972,7 +3008,7 @@
 		67186B3A1EB248F100CFAFFB /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				APPLICATION_EXTENSION_API_ONLY = YES;
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
revert

Reply:
should be able to compile for extension as well

Reply:
But it shouldn't
![image](https://user-images.githubusercontent.com/20190113/53801996-ba2e2d00-3f51-11e9-9d6f-dc5493d1ac80.png)